jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java

```diff
@@ -9,11 +9,13 @@
 
 
 public class HomePageEx extends PageBase {
-    public HomePageEx() { super("/index.html", "Home");}
+
     @FindBy(css="form #name") public WebElement userName;
     public WebElement password = $("form #password");
-
+    @UI("#login-button") public WebElement enterUI;
```

Comment:
I think it's better to leave this line blank and return this line to 18. More readable in my opinion =)

Reply:
done:

public class HomePageEx extends PageBase {

    @FindBy(css="form #name") public WebElement userName;
    public WebElement password = $("form #password");

    @UI("form #name") public UIElement userNameUI;
    public UIElement passwordUI = $("form #password");

    @UI("#login-button") public WebElement enterUI;
    public HomePageEx() { super("/index.html", "Home");}
}